.github/workflows/prerelease.yaml

```diff
@@ -0,0 +1,27 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: pre-release
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  default:
+    strategy:
+      fail-fast: false
+      matrix:
+        distro: [foxy, galactic, rolling]
+
+    env:
+      ROS_DISTRO: ${{ matrix.distro }}
+      PRERELEASE: true
+      BASEDIR: ${{ github.workspace }}/.work
+
+    name: "${{ matrix.distro }}"
```

Comment:
Prefix with "Prerelease Test" or so?

Reply:
Sure.

Reply:
For the convention I can write this to all repos or left it like that. GHA Already kinda adds this though: 
![image](https://user-images.githubusercontent.com/13888103/122269605-154b7a00-cee6-11eb-9658-a39b4a0e675f.png)

Reply:
If gha already adds something, then it's fine like this. I just noticed that the checks for this PR only list the distro names.